### PR TITLE
Replaced python sdk usage with docker cli

### PR DESCRIPTION
**Pull request by hpohekar**
Replaced python sdk usage with docker cli in test_fluent_session.py.

image

### CODE REVIEW

File: tests/test_fluent_session.py [hunk 1]
```diff
@@ -54,18 +54,27 @@ def print_transcript(transcript):
     assert not print_transcript.called
 
 
+def get_container_ids_set():
+    proc = subprocess.Popen(["docker", "ps", "-q"], stdout=subprocess.PIPE)
+    output_bytes = proc.stdout.read()
+    output_str = output_bytes.decode()
```
Comment (mkundu1) [inline tests/test_fluent_session.py:60]:
```
ids = output_str.strip().split()
return set(ids)
```

**Comment by mkundu1** [comment]
Please also remove the docker dependency from requirements\requirements_tests.txt.

File: tests/test_fluent_session.py (hunk 1 above)
Comment (hpohekar) [reply tests/test_fluent_session.py:60]: Done. Thank you.

**Comment by hpohekar** [comment]
> Please also remove the docker dependency from requirements\requirements_tests.txt.

Done. Thank you.

**Review by mkundu1 — APPROVED** [review]

**Review by prmukherj — APPROVED** [review]

**Pull request merged**